src/main/java/soolutions/hometaskapp/task/Importance.java

```diff
@@ -0,0 +1,8 @@
+package soolutions.hometaskapp.task;
+
+public enum Importance {
```

Comment:
NIP: Should we consider another name for this like `Priority`?

Reply:
Priority is about comparative ranking. 
Importance is about intrinsic value. 

I chose the word importance, because for a person with a recurring task (say "Washing Dishes"), that may be their only task. Furthermore, all recurring tasks are non-optional -- either they are completed, or they are outsourced to another worker. Therefore, it is not about whether or not they should be, nor about in which order they should be done. 

It is simply about how valuable it is for this Task to be done. This impacts the funding rates. Critical tasks cost more and pay more than tasks of low importance.